src/Psr18ClientDiscovery.php

```diff
@@ -24,7 +24,7 @@ public static function find()
         try {
             $client = static::findOneByType(ClientInterface::class);
         } catch (DiscoveryFailedException $e) {
-            throw new \Http\Discovery\Exception\NotFoundException('No PSR-18 clients found. Make sure to install a package providing "psr/http-client-implementation". Example: "php-http/guzzle6-adapter".', 0, $e);
+            throw new \Http\Discovery\Exception\NotFoundException('No PSR-18 clients found. Make sure to install a package providing "psr/http-client-implementation". Example: "php-http/guzzle7-adapter".', 0, $e);
```

Comment:
Just a query here on deps; `php-http/guzzle6-adapter` is compatible with PHP 7.1 (as `php-http/discovery` is itself), but `php-http/guzzle7-adapter` isn't. This change would now recommend `php-http/guzzle7-adapter` when PHP 7.1 users cannot actually install this?

Reply:
it is not related to the fix, just stumbled over it when reading the code. it is an example, not a recommendation.